### PR TITLE
Use mixins for cache and logger

### PR DESCRIPTION
**Pull request by alex-deaconu**
Update unit tests

### CODE REVIEW

**Comment by alex-deaconu** [comment]
Fixes for the other PR.

@ezequielc please review. Thanks!

File: test/unit/rise-data-weather.html [hunk 1]
```diff
@@ -40,23 +38,22 @@
...
 
   suite("rise-data-weather", () => {
     let sandbox = sinon.createSandbox();
     let element;
+    let cache, logger;
 
     setup(() => {
+      element = fixture("test-block");
+
+      cache = element.__proto__.__proto__;
```
Comment (ezequielc) [inline test/unit/rise-data-weather.html]: @alex-deaconu For testing the Weather `element`, maybe you don't need to reference `cache` and `logger`. You could just stub `puCache`, `getCache` in the `element` itself.  Thanks

File: src/cache.js [hunk 2]
```diff
@@ -1,77 +1,79 @@
...
+  CACHE_NAME: "rise-data-weather",
+  CACHE_DURATION: 1000 * 60 * 60 * 2,
+};
 
-function getCache() {
-  if ( caches && caches.open ) {
-    return caches.open( CACHE_CONFIG.CACHE_NAME );
-  } else {
-    logger.log( "warning", "cache API not available" );
+export function cache( superClass ) {
```
Comment (ezequielc) [inline src/cache.js]: @alex-deaconu I would prefer to use CamelCase for class names for consistency with Polymer. Thanks!

File: test/unit/rise-data-weather.html (hunk 1 above)
Comment (ezequielc) [reply test/unit/rise-data-weather.html]: We talked. My suggestion doesn't work.

**Pull request merged**